bundles/framework/layerlist/model/LayerGroup.js

```diff
@@ -124,8 +142,9 @@ export class LayerGroup {
         }
         return terms.every(key => searchableIndex.indexOf(key.toLowerCase()) !== -1);
     }
+
     clone () {
-        const clone = new LayerGroup(this.id, this.groupMethod, this.name, this.description, this.parentId);
+        const clone = new LayerGroup(this.id, this.type, this.name, this.description, this.parentId);
```

Comment:
Was the second parameter changed by accident or do we have the type in both groupMethod and type variables?

Reply:
accident. fixed